modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About external IP addresses
+
+You can allow external traffic into the cluster by defining an *external IP* configuration.
```

Comment:
A better wording might be something like:

One way to make Kubernetes Services reachable by external clients is with the use of External IPs. External IPs are an additional virtual IP assigned to a Service that does not need to be inside the cluster's ServiceCIDR. External IPs are, generally, most useful for clusters in a bare-metal environment.

ExternalIPs function similarly to NodePort services, where traffic directed to a local node will be accepted and load-balanced. However, the administrator needs to configure their routing environment to ensure ExternalIPs are routed to the OpenShift cluster.

Reply:
Also, somewhere add:

ExternalIPs can be a security risk, because in-cluster traffic to an ExternalIP are directed to that service. This could allow cluster users to intercept sensitive traffic destined for external resources. Therefore, OpenShift, by default, disables use of ExternalIPs. See below for how to enable limited ExternalIPs.

Reply:
@squeed, does an External IP need not (but can) be in the ServiceCIDR, or is there a collision risk if it is? Can IP addresses automatically assigned from the ServiceCIDR co-exist with an External IP range?

Thanks.